ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4409,7 +4417,130 @@ HWDDC:
   implements:
   - CONTROL
 
+# New abstracts for MFA BV100 HVAC
+CFDPM:
+  description: "Carbon filter pressure monitoring."
+  is_abstract: true
+  uses:
+  - carbon_filter_differential_pressure_sensor
+  implements:
+  - MONITORING
+ 
+VOADM2X:
+  description: "Variable outside air damper monitoring."
+  is_abstract: true
+  opt_uses:
+  - economizer_mode
+  - mixed_air_temperature_sensor
+  - outside_air_damper_percentage_sensor_1
+  - outside_air_damper_percentage_sensor_2
+  - low_limit_outside_air_damper_percentage_command
+  - outside_air_flowrate_sensor_1
+  - outside_air_flowrate_sensor_2
+  uses:
+  - outside_air_temperature_sensor
+  - outside_air_damper_percentage_command_1
+  - outside_air_damper_percentage_command_2
+  implements:
+  - MONITORING
+ 
+EHHRC:
+  description: "Exhaust hydronic heat recovery coil with an isolation valve."
+  is_abstract: true
+  uses:
+  - heat_recovery_water_isolation_valve_command
+  - leaving_heat_recovery_coil_temperature_sensor
+  - entering_heat_recovery_coil_temperature_sensor
+  - exhaust_air_flowrate_sensor
+  implements:
+  - MONITORING
+ 
+DPBHCC:
+  description: "Two-pipe binary (open/closed) heating and cooling control. Has a single control valve
+  that is fed by two separate headers for heating and cooling water. There is an isolation valve for
+  each incoming system and a single control valve. Valve and mode control to zone temperature 
+  (heating/cooling setpoint configuration)."
+  is_abstract: true
+  opt_uses:
+  - cooling_request_count
+  - heating_request_count
+  - zone_air_relative_humidity_sensor
+  uses:
+  - chilled_supply_water_isolation_valve_command
+  - chilled_supply_water_isolation_valve_status
+  - heating_supply_water_isolation_valve_command
+  - heating_supply_water_isolation_valve_status
+  - zone_air_cooling_temperature_setpoint
+  - zone_air_heating_temperature_setpoint
+  - zone_air_temperature_sensor
+  implements:
+  - CONTROL
+ 
+FTC:
+  description: "Floor temperature control"
```

Comment:
please add a bit more here: "[...] where the temperatures are embedded in the floor (as opposed to open to the air)."